File: LeetCode/Array/Python/FizzBuzz.py
def fizz_buzz(n):
    result = []

    for i in range(1, n+1):
        if i % 15 == 0:
            result.append("FizzBuzz")
        elif i % 5 == 0:
            result.append("Buzz")
        elif i % 3 == 0:
            result.append("Fizz")
        else:
            result.append(str(i))

    return result

def main():
	return_list = fizz_buzz(15)

	for item in return_list:
		print(item)

if (__name__ == "__main__"):
	main()

File: LeetCode/Array/C++/FizzBuzz.cpp
#include <vector>
#include <string>
#include <iostream>

std::vector<std::string> fizzBuzz(int n) {
    std::vector<std::string> result;

    for (int i = 1; i <= n; ++i) {
        if (i % 15 == 0) {
            result.emplace_back("FizzBuzz");
        } else if (i % 5 == 0) {
            result.emplace_back("Buzz");
        } else if (i % 3 == 0) {
            result.emplace_back("Fizz");
        } else {
            result.emplace_back(std::to_string(i));
        }
    }

    return result;
}

int main()
{
    std::vector<std::string> returnList = fizzBuzz(15);

    for(auto item : returnList)
    {
        std::cout << item << "\n";
    }
}